src/services/measure/measurearea.js

```diff
@@ -0,0 +1,84 @@
+goog.provide('ngeo.interaction.MeasureArea');
+
+goog.require('ngeo.interaction.Measure');
+goog.require('ol.geom.Polygon');
+goog.require('ol.interaction.Draw');
+
+
+
+/**
+ * @classdesc
+ * Interaction dedicated to measure length.
+ *
+ * @constructor
+ * @extends {ngeo.interaction.Measure}
+ * @param {{
+ *    startMsg: (string|undefined),
+ *    continueMsg: (string|undefined),
+ *    style: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *    sketchStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined)
+ * }=} opt_options Options
+ * @export
```

Comment:
We don't want to use `@export` here, because we don't want/need the constructor to be exported when compiling ngeo and app together. Add a file to the `exports` dir instead, with explicit `goog.export*` statements.